### ISSUE TITLE
[Treantjs]: Is a chart created using treant.js able to be made responsive?

### ISSUE THREAD

**Issue by egor-progger** [egor-progger/pointlinejs#125]
### Description from treantjs issue

> I need to make my chart responsive as soon as I can. Is it possible?

### Treantjs issue link

https://github.com/fperucic/treant-js/issues/159

**Comment by ETJamesDev** [comment]
Yes, this is possible with additional CSS/JavaScript adjustments!

Want me to fork this?

**Comment by egor-progger** [comment]
@ETJamesDev , good. May some disscusion before start? I only copy issue from treantjs. I read source issue, and I didn't see idea of responsive from author issue. Are you going to add auto scroll when user change browser's window size?
